dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategorySQL.java

```diff
@@ -40,5 +40,9 @@ public String getSortedChildren() {
 				"category.inode = tree.child and tree.parent = ? and category_1_.inode = category.inode " +
 				" and category_1_.type = 'category'";
 	}
+	
+	public String getVelocityVarNameCount() {
+	    return "select count(*) as test from category where category_velocity_var_name like ?";
+	}
```

Comment:
I think they use a method in order to be override by a subclass, such as OracleCategorySQL
However the constants still valid in this context, the method will return the constant.